content/python/concepts/math-module/terms/math-pow/math-pow.md

```diff
@@ -45,3 +45,13 @@ print(math.pow(5.5, 3.3))
 
 # Output: 277.457759723262
 ```
+
+## Codebyte Example
+
+Return the value of 4 raised to the power of 3
+
+```codebyte/py
+import math
+
+print(math.pow(4, 3))
```

Comment:
```suggestion
print(math.pow(2, 0.0))
print(math.pow(1.0, 2))
```

Reply:
I updated this example to illustrate one of the special cases in order to differentiate it more from the other examples